tests/utils/http_requests.py

```diff
@@ -11,12 +11,9 @@
 
 
 class HttpRequests(object):
-    try:
-        requests.packages.urllib3.disable_warnings()
-    except AttributeError:
-        # Running on OSX the disable_warnings is not available, so catch and deal with that. The effect of
-        # ignoring the error is additional warnings so no real risk.
-        pass
+    import urllib3
+
+    urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
Is this something that we could do at the application level?

Reply:
We could probably do it with environment vars, but given this only really runs on a linux host and has hardcoded RPM deps that need to be fixed this way, I'd rather do it in code.